.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
                 "run",
                 "Admin.py"
             ],
-            "cwd": "${workspaceFolder}/code/admin",
```

Comment:
When we are using azd an .env file is generated under the .azure folder - do we have to add something like
```
"envFile": "${workspaceFolder}/.azure/dev/.env"
```
to each of these launch - or is there an azd way of doing this?

Reply:
Usually we don't do this. The parameters in the `.env` file generated under the `.azure` folder are not enough to deploy the application. For security reasons, all keys will not appear in this file. (Except for one case: when you use `Key Vault`, all Keys at this time are just KeyName, so we can output them)

Reply:
Will pick this up in #69